platform/xbox/functions/math/pow.c

```diff
@@ -2,15 +2,24 @@
 
 double pow(double x, double y)
 {
-    return exp(y*log(x));
+    if (x < 0.0 && y != trunc(y)) {
+        return -NAN;
+    }
+    return exp(y*log(fabs(x)));
```

Comment:
Taking the absolute of `x` looks wrong.
I did not confirm against the C spec or another libc, but I'd assume that:

- `(-1) ^ 1` should be `-1`
- `(-1) ^ 2` should be `+1`
- `(-1) ^ 3` should be `-1`
- ...

but your implementation would be using:

- `abs(-1) ^ 2` will be `+1`
- `abs(-1) ^ 2` will be `+1` (this would be incorrect)
- `abs(-1) ^ 2` will be `+1`
- ...

Reply:
Oops. I didn't test the combination of a negative base with an odd exponent. I now pushed implementations for `copysign` and `remainder` and used them to fix this case.